frontend/app/component/form/candidates_votes_form/CandidatesVotesForm.test.tsx

```diff
@@ -1,5 +1,22 @@
+/** 
+ * Test Plan
```

Comment:
I'd prefer to remove the Test Plan comments in favor of (1) moving the use case document with errors and warnings here; (2) making that document easier to read / get an overview from.

Reply:
removed